addons/hitreactions/functions/fnc_checkWeaponDrop.sqf

```diff
@@ -0,0 +1,32 @@
+#include "..\script_component.hpp"
+/*
+ * Author: KJW
+ * Checks if the unit should drop their weapon based on projectile hit info.
+ *
+ * Arguments:
+ * 0: Surface type <STRING>
+ * 1: Components <ARRAY>
+ * 2: Position <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ["", [], [1, 1, 1]] call ace_hitreactions_fnc_checkWeaponDrop
+ *
+ * Public: No
+ */
+
+params ["_surfaceType", "_components", "_position"];
+
+private _didHitArm = _components findAny GVAR(armComponents) isNotEqualTo -1;
```

Comment:
For readability's sake:
```suggestion
private _didHitArm = _components findAny GVAR(armComponents) != -1;
```

Reply:
Committed however `!=` is slower than `isNotEqualTo` in some cases. Don't think that is the case in this one.